ui-admin/src/study/kits/KitList.tsx

```diff
@@ -42,16 +105,43 @@ export default function KitList({ studyEnvContext }: { studyEnvContext: StudyEnv
     loadKits()
   }, [studyEnvContext.study.shortcode, studyEnvContext.currentEnv.environmentName])
 
+  const kitsByStatus = _groupBy(kits, kit => {
+    return statusTabs.find(tab => tab.status === kit.pepperStatus?.currentStatus)?.status || 'ERROR'
+  })
+
   return <LoadingSpinner isLoading={isLoading}>
-    <KitListView kits={kits} studyEnvContext={studyEnvContext}/>
+    <Tabs
```

Comment:
I don't know of any reason to use bootstrap tabs if you're going to do mountOnEnter/unMountOnExit.  is there an accessibility benefit?  Otherwise, the code will be cleaner to just manage it ourselves